src/constants/ContentSectionAnchors.js

```diff
@@ -12,45 +12,45 @@ import { SectionKind } from 'docc-render/constants/PrimaryContentSection';
 
 export const MainContentSectionAnchors = {
   topics: {
-    title: 'Topics',
+    title: 'sections.topics',
     anchor: 'topics',
     level: 2,
   },
   defaultImplementations: {
-    title: 'Default Implementations',
+    title: 'sections.default-implementations',
     anchor: 'default-implementations',
     level: 2,
   },
   relationships: {
-    title: 'Relationships',
+    title: 'sections.relationships',
     anchor: 'relationships',
     level: 2,
   },
   seeAlso: {
-    title: 'See Also',
+    title: 'sections.see-also',
     anchor: 'see-also',
     level: 2,
   },
 };
 
 export const PrimaryContentSectionAnchors = {
   [SectionKind.declarations]: {
-    title: 'Declaration',
+    title: 'sections.declarations',
```

Comment:
```suggestion
    title: 'sections.declaration',
```

Reply:
The key is plural in the JSON right now (but maybe it should be singular to match the text)

Reply:
Right! That's what I meant

Reply:
True! fixed